server/default.nix

```diff
@@ -0,0 +1,17 @@
+let
+  config = {
+    packageOverrides = pkgs: rec {
+      haskellPackages = pkgs.haskellPackages.override {
+        overrides = haskellPackagesNew: haskellPackagesOld: rec {
+          icepeak = haskellPackagesNew.callPackage ./server.nix { };
+          websockets = haskellPackagesNew.callPackage ../nix/websockets.nix { };
+        };
+      };
+    };
+  };
+
+  pkgs = import ../nix/nixpkgs.nix { inherit config; };
```

Comment:
Would be nice if this were a parameter (the import can be its default) value so that you can use Icepeak from another project with a different nixpkgs if desires.

Reply:
I attempted something in 8c94826aa24b7616cf8bf51a441d8d9e19193bbf, but I'm not sure that's the best way to do this.

Is there a way to pass in new config / package overrides on an already imported set of nixpkgs?

Reply:
Not sure. There is no overrideAttrs on the object returned by Nixpkgs.